_posts/fr/newsletters/2020-12-23-newsletter.md

```diff
@@ -0,0 +1,956 @@
+[complexités][bcc19569]---était d'annoncer les nouvelles transactions en
+utilisant leur txid de témoin (wtxid). Moins d'un mois après l'ajout du
+BIP339 au dépôt des BIP, les annonces de wtxid ont été [fusionnées][bcc18044]
+dans Bitcoin Core. Bien qu'il s'agisse apparemment d'un problème mineur
+sans effet évident sur les utilisateurs, les annonces wtxid simplifient
+le développement des mises à jour espérées, telles que le [relai de paquet][topic package relay].
+
+## Août
+
+{:#signet}
+Après [plus][bips900] d'un [an][bips983] de [développement][default signet
+discussion], y compris de multiples changements basés sur le feedback, la
+[dernière révision majeure][bips947] du [BIP325][] de la spécification de
+[signet][topic signet] a été fusionnée début août. Signet est un protocole
+qui permet aux développeurs de créer des réseaux de test publics et également
+le nom du principal signet public. Contrairement au réseau de test public
+existant de Bitcoin (testnet), les blocs signet doivent être signés par une
+partie de confiance. Cela permet d'éviter le vandalisme et d'autres problèmes
+qui se produisent parce que testnet utilise le mécanisme de convergence du
+consensus basé sur l'économie de Bitcoin (preuve de travail), même si les
+pièces de testnet n'ont aucune valeur. La possibilité d'activer optionnellement
+le signet a finalement été [ajoutée][bcc18267] à Bitcoin Core en septembre.
+
+{:#anchor-outputs}
+Près de deux ans après que Matt Corallo ait [proposé pour la première fois][news24 cpfp carve out]
+le [mécanisme d'exclusion du CPFP][topic cpfp carve out],
+la spécification LN a été [mise à jour][news112 bolts688] pour permettre la
+création de [sorties d'ancrage][topic anchor outputs] qui utilisent les exclusions
+pour la sécurité. Les sorties d'ancrage permettent à une transaction multipartite
+de faire l'objet d'un prélèvement de frais même si l'une des parties tente
+d'utiliser une attaque de type [épinglage de transaction][topic transaction pinning]
+pour empêcher le prélèvement de frais. La possibilité de faire monter les frais
+des transactions même dans des conditions adverses permet aux nœuds LN d'accepter
+des transactions hors chaîne sans s'inquiéter de l'augmentation des frais à l'avenir.
+Si, plus tard, il devient nécessaire de diffuser cette transaction hors chaîne,
+le nœud peut choisir un taux de frais approprié au moment de la diffusion. Cela
+simplifie le protocole LN et améliore plusieurs aspects de sa sécurité.
+
+<div markdown="1" class="callout" id="optech">
+### Sommaire 2020<br>Bitcoin Optech
+
+Au cours de la troisième année d'existence d'Optech, 10 nouvelles sociétés
+membres se sont jointes à nous<!-- Veriphi, SwanBitcoin, YellowCardIO,
+Xbtogroup, Bitonic, Fidelity Center for Applied Technology, AndgoInc, BTSEcom,
+EdgeWallet, Bitbank_inc -->, nous avons organisé un [atelier taproot][] à
+Londres, publié 51 bulletins d'information hebdomadaires< ! -- 78 à 129 -->,
```

Comment:
< ! -- 78 à 129 --> shouldn't render as it should be an HTML comment, can you reformat so this doesn’t render t the screen?

Reply:
Hello @bitschmidty ,

I make change, but now I have an error 
![image](https://user-images.githubusercontent.com/4174888/217594670-26c0fc3b-1238-473e-a6d4-72822af7443e.png)

Reply:
OK I think it's OK now,

can you review please @bitschmidty  ?

Reply:
Hello @bitschmidty can you check if it's OK please ?

I would create PR for newsletter but I need to close this one to rebase my repository as discussed

thanks,